net/wget/patches/100-fix-hsts-time.patch

```diff
@@ -0,0 +1,45 @@
+--- a/src/hsts.c
```

Comment:
This patch is backported from upstream? If not, are you going to send it to them? It is not clear. There is missing commit header (author, commit messace, etc.)

Reply:
https://savannah.gnu.org/bugs/index.php?61451